packages/react-native-gesture-handler/src/v3/detectors/ReanimatedNativeDetector.tsx

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useMemo, useRef } from 'react';
+import {
+  NativeEventsManager,
+  Reanimated,
+} from '../../handlers/gestures/reanimatedWrapper';
+import HostGestureDetector, {
+  type RNGestureHandlerDetectorNativeComponentProps,
+} from './HostGestureDetector';
+import { findNodeHandle } from 'react-native';
+
+let NativeEventsManagerImpl = Reanimated?.NativeEventsManager;
+
+if (!NativeEventsManagerImpl) {
+  // Fallback to empty object when Reanimated or NativeEventsManager is not available
```

Comment:
Maybe I've missed something, but is there a fallback to empty object?

Reply:
I'm not even sure who hallucinated this, fixed in [df7eaac](https://github.com/software-mansion/react-native-gesture-handler/pull/3837/commits/df7eaac45bf4de25b968a6a16dbf87bf958bd6fa).